difflame

```diff
@@ -1412,6 +1412,9 @@ except:
     traceback.print_exc()
     sys.exit(1)
 
+if not diff_output:
```

Comment:
Maybe tell something to the user so they know what is going on?

Reply:
I was trying to mimic default `git diff` behaviour in this case. E.g. `git diff 9c91630 README.md` has empty input.
I've added a note to make it more explicit.

Reply:
Yeah, that's also valid. Let me think about it fir a little bit.... just in case, please, squash both commits, I don't see a need to stack them given that the changes are very small (and related to the same thing).

Reply:
I think writing something on `sys.stderr` would not be something to complain about.... but let me think about it.